tools/encoder/src/EncoderArguments.h

```diff
@@ -20,6 +20,7 @@ class EncoderArguments
         FILEFORMAT_UNKNOWN,
         FILEFORMAT_DAE,
```

Comment:
Can you remove the FILEFORMAT_DAE while you are at it. We no longer support this.